templates/tfengine/recipes/org/devops.hcl

```diff
@@ -0,0 +1,84 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+schema = {
+  title = "Devops Recipe"
+  required = [
+    "project_id",
+    "org_id",
```

Comment:
org_id shouldn't be required as folder_id can be specified instead of org_id

Why are we changing parent_id to org_id + folder_id?

Reply:
The reason was parent_id was just duplicating the org_id/folder_id, etc that was already set in the config.